tests/components/fronius/test_config_flow.py

```diff
@@ -0,0 +1,227 @@
+"""Test the fronius config flow."""
+from unittest.mock import patch
+
+from pyfronius import FroniusError
+
+from homeassistant import config_entries, setup
+from homeassistant.components.fronius.const import DOMAIN
+from homeassistant.const import CONF_HOST
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import (
+    RESULT_TYPE_ABORT,
+    RESULT_TYPE_CREATE_ENTRY,
+    RESULT_TYPE_FORM,
+)
+
+from tests.common import MockConfigEntry
+
+INVERTER_INFO_RETURN_VALUE = {"inverters": [{"unique_id": {"value": "1234567"}}]}
+LOGGER_INFO_RETURN_VALUE = {"unique_identifier": {"value": "123.4567"}}
+
+
+async def test_form_with_logger(hass: HomeAssistant) -> None:
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
```

Comment:
We don't need to set up the persistent notification integration anymore. All of these can be removed.